docs/manual.md

```diff
@@ -538,48 +586,56 @@ Next$1 == year - 80 >= 18 /\ hasLicense' <- TRUE /\ (year' <- year)
 
 As you can see, the model checker did two things:
 
- 1. It has translated several expressions that look like `x' = e` into `x' <- e`.
-   For instance, you can see `year' <- 80` and `hasLicense' <- FALSE` in `Init$0`.
-   We call these expressions **assignments**.
-
- 1. It has replaced the operator `Next` into two operators `Next$0` and `Next$1`.
+1. It has translated several expressions that look like `x' = e` into `x' <- e`.
+   For instance, you can see `year' <- 80` and `hasLicense' <- FALSE` in
+   `Init$0`. We call these expressions **assignments**.
+1. It has factored the operator `Next` into two operators `Next$0` and `Next$1`.
    We call these operators **symbolic transitions**.
 
-The pure TLA+ does not have the notions of assignments and symbolic
+Pure TLA+ does not have the notions of assignments and symbolic
 transitions.  However, TLC sometimes treats expressions `x' = e` and `x' \in S`
 as if they were assigning a value to the variable `x'`. TLC does so
 dynamically, during the breadth-first search. Apalache looks statically for assignments
 among the expressions `x' = e` and `x' \in S`.
 
+<!-- TODO(shonfeder): The first stentence of the next passage isn't clear to me.
+     It doesn't seem to be exemplified in the example code at all, and no further
+     example of how this is provided. -->
 Moreover, Apalache splits action operators `Init` and `Next` into disjunctions
 (e.g., `A_1 \/ ... \/ A_k`). The main contract between the assignments and
 symbolic transitions is as follows:
 
-> For every variable `x` declared with `VARIABLE`, there is exactly one assignment
-of the form `x' <- e` in every symbolic transition `A_j`.
+> For every variable `x` declared with `VARIABLE`, there is exactly one
+> assignment of the form `x' <- e` in every symbolic transition `T$n`.
```

Comment:
Now that I'm down to reading the theory section, I think this change is incorrect. IIUC, we *are* using `A_i` to represent the `i`th step in a symbolic transition, which happens to have the concrete syntax `A$i`. I'll change this back, and instead add a note to the preceding paragraph which informs the reader that we'll use `A_i` in this way.